contrib/rhel/build-netdata-rpm.sh

```diff
@@ -13,24 +13,33 @@ run autoreconf -ivf
 run ./configure --enable-maintainer-mode
 run make dist
 
-version=$(grep PACKAGE_VERSION < config.h | cut -d '"' -f 2)
-if [ -z "${version}" ]
-then
-    echo >&2 "Cannot find netdata version."
-    exit 1
+typeset version="$(grep PACKAGE_VERSION < config.h | cut -d '"' -f 2)"
+if [[ -z "${version}" ]]; then
+	run_failed "Cannot find netdata version."
```

Comment:
Nit: You _haven't_ actually changed anything here. And I've more commonly seen/used `declare` anyway. But can we revert this bit, Its a bit superfluous? Also please prefer as much POSIX-sh as you can 👍